packages/components/src/components/crypto-money/crypto-money.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+
+const CryptoMoney = ({
+    realValue,
+    toggleValue,
+}) => {
+    const [cryptoToggleView, setCryptoToggleView] = useState(true);
+
+    const toggleCryptoValue = (e) => {
+        e.preventDefault();
+        setCryptoToggleView(!cryptoToggleView);
+    };
+
+    return (
+        <span onClick={toggleCryptoValue}>
```

Comment:
we don't want to change the value of the element, we want onmouseenter and onmouseleave, you can see how the other columns in the table have onhover effect to show a tooltip, like the trade type icons